backend/collaboration-service/app/src/constants/question.ts

```diff
@@ -16,4 +18,6 @@ export const TOPIC_TAG = {
     MATH: "Math",
     DIVIDE_AND_CONQUER: "Divide and Conquer",
     DYNAMIC_PROGRAMMING: "Dynamic Programming",
-} as const;
\ No newline at end of file
+} as const;
```

Comment:
Do we still need these hardcoded?

Reply:
Nope but I haven't integrated your stuff yet